cpu/stm32l1/stmclk.c

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_stm32l1
+ * @{
+ *
+ * @file
+ * @brief       Implementation of the kernel cpu functions
+ *
+ * @author      Thomas Eichinger <thomas.eichinger@fu-berlin.de>
+ * @author      Nick van IJzendoorn <nijzendoorn@engineering-spirit.nl>
+ * @author      Víctor Ariño <victor.arino@zii.aero>
+ * @author      Oleg Artamonov <info@unwds.com>
+ *
+ * @}
+ */
+
+#include <string.h>
+#include "cpu.h"
+#include "stmclk.h"
+#include "periph_conf.h"
+
+/* See if we want to use the PLL */
+#if defined(CLOCK_PLL_DIV) || defined(CLOCK_PLL_MUL) || \
+defined(CLOCK_PLL_DIV_HSE) || defined(CLOCK_PLL_MUL_HSE) || \
+defined(CLOCK_PLL_DIV_HSI) || defined(CLOCK_PLL_MUL_HSI)
```

Comment:
You aligned the `#if` statement, but the subsequent defines which belong to the same statement are also aligned between them, thus:

```c
#if defined(CLOCK_PLL_DIV) || defined(CLOCK_PLL_MUL) || \
    defined(CLOCK_PLL_DIV_HSE) || defined(CLOCK_PLL_MUL_HSE) || \
    defined(CLOCK_PLL_DIV_HSI) || defined(CLOCK_PLL_MUL_HSI)
```
is the result we want.